src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -30,6 +30,41 @@ static void AppendAlterDatabaseOwnerStmt(StringInfo buf, AlterOwnerStmt *stmt);
 static void AppendAlterDatabaseStmt(StringInfo buf, AlterDatabaseStmt *stmt);
 static void AppendDefElemConnLimit(StringInfo buf, DefElem *def);
 
+const DefElemOptionFormat create_database_option_formats[] = {
+	{ "owner", " OWNER %s", OPTION_FORMAT_STRING },
+	{ "template", " TEMPLATE %s", OPTION_FORMAT_STRING },
+	{ "encoding", " ENCODING %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "strategy", " STRATEGY %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "locale", " LOCALE %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "lc_collate", " LC_COLLATE %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "lc_ctype", " LC_CTYPE %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "icu_locale", " ICU_LOCALE %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "icu_rules", " ICU_RULES %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "locale_provider", " LOCALE_PROVIDER %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "collation_version", " COLLATION_VERSION %s", OPTION_FORMAT_LITERAL_CSTR },
+	{ "tablespace", " TABLESPACE %s", OPTION_FORMAT_STRING },
+	{ "allow_connections", " ALLOW_CONNECTIONS %s", OPTION_FORMAT_BOOLEAN },
+	{ "connection_limit", " CONNECTION LIMIT %d", OPTION_FORMAT_INTEGER },
+	{ "is_template", " IS_TEMPLATE %s", OPTION_FORMAT_BOOLEAN },
+	{ "oid", " OID %d", OPTION_FORMAT_OBJECT_ID }
+};
+
+/*
+ * DeparseAlterDatabaseOwnerStmt
```

Comment:
https://github.com/citusdata/citus/pull/7253#discussion_r1383023307

> We prefer camel-case over snake-case (e.g., prefix_appended_for_basic_options)
> * Functions start with a comment (except AppendFoo ones)
>  /\*
>   \* PreprocessAlterDatabaseRenameStmt ..
>   \*/
> * We avoid from doing unnecessary style changes in PRs (adding deleting a newline etc.)
> * Functions names start with a "Capital" letter.

As I noted earlier, we don't need to have function header comments for Append / Deparse functions